crates/stackable-operator/src/commons/authentication/oidc.rs

```diff
@@ -17,19 +17,27 @@ use crate::commons::{
 
 pub type Result<T, E = Error> = std::result::Result<T, E>;
 
-pub const DEFAULT_OIDC_WELLKNOWN_PATH: &str = ".well-known/openid-configuration";
 pub const CLIENT_ID_SECRET_KEY: &str = "clientId";
 pub const CLIENT_SECRET_SECRET_KEY: &str = "clientSecret";
 
+const DEFAULT_OIDC_WELLKNOWN_PATH: &str = "./.well-known/openid-configuration";
```

Comment:
```suggestion
const DEFAULT_WELLKNOWN_OIDC_CONFIG_PATH: &str = "/.well-known/openid-configuration";
```

This also needs updating in the `https_endpoint_url` unit test.

Reply:
Attention: You can not have a leading slash, that messes everything up ^^
But yeah, the `./` was a testing leftover, removed it again